### PR TITLE
Assembly: Fix lag during dragging of large assemblies.

### PR DESCRIPTION
**Pull request by PaddleStroke**
- Bundle fix joints together when dragging to make things simpler for solver. 
 - Show and redraw only the movingJoint during dragging. 
 - Do not recompute joints during dragging.

This together make the dragging steps much faster.

Fix https://github.com/FreeCAD/FreeCAD/issues/16771

### CODE REVIEW

File: src/Mod/Assembly/App/AssemblyObject.cpp [hunk 1]
```diff
@@ -142,13 +143,14 @@ App::DocumentObjectExecReturn* AssemblyObject::execute()
     return ret;
 }
 
-int AssemblyObject::solve(bool enableRedo, bool updateJCS)
+int AssemblyObject::solve(bool enableRedo, bool updateJCS, bool bundleFixedJts)
```
Comment (chennes) [inline src/Mod/Assembly/App/AssemblyObject.cpp:146]: We've already got significant boolean blindness problems with this method: is there another approach that would make sense here? The classic solution is a set of enumerations.

Comment (kadet1090) [reply src/Mod/Assembly/App/AssemblyObject.cpp:146]: Essentialy this is bit flags, so I'd replace it with bit flags kind of enum. So `AssemblySolve::EnableRedo | AssemblySolve::BundleFixedJts` as replacement for `true, false, true`

Comment (PaddleStroke) [reply src/Mod/Assembly/App/AssemblyObject.cpp:146]: Actually this argument is not needed it's cleaner without it so I'll just remove it

**Comment by yorikvanhavre** [comment]
backported

**Comment by GAZ082** [comment]
@chennes is this in RC2? because assy performance is horrible in RC2 but acceptable in 1.1.dev

**Pull request merged**